azure-mgmt-recoveryservicesbackup/azure/mgmt/recoveryservicesbackup/models/azure_backup_server_container.py

```diff
@@ -72,13 +59,12 @@ class AzureBackupServerContainer(ProtectionContainer):
         'registration_status': {'key': 'registrationStatus', 'type': 'str'},
         'health_status': {'key': 'healthStatus', 'type': 'str'},
         'container_type': {'key': 'containerType', 'type': 'str'},
-        'protectable_object_type': {'key': 'protectableObjectType', 'type': 'str'},
         'can_re_register': {'key': 'canReRegister', 'type': 'bool'},
         'container_id': {'key': 'containerId', 'type': 'str'},
         'protected_item_count': {'key': 'protectedItemCount', 'type': 'long'},
         'dpm_agent_version': {'key': 'dpmAgentVersion', 'type': 'str'},
-        'dpm_servers': {'key': 'DPMServers', 'type': '[str]'},
-        'upgrade_available': {'key': 'UpgradeAvailable', 'type': 'bool'},
+        'dpm_servers': {'key': 'dpmServers', 'type': '[str]'},
```

Comment:
Service endpoints are case-insensitive, though these will be the actual cases returned by service. 
Btw how deserialization is handled in msrest? Is it case-sensitive?

Reply:
Deserialization is case-sensitive in all SDK language except C#.... This is the expected behavior for a JSON return (JSON is case-sensitive).
We have trouble currently because:
- C# doesn't care about case sensitivity
- The linter screams if this is not camelCase, mixing-up promoting good behavior (when you still can change your server) and asking Swagger change. If your server is already impossible to change, the right fix is not to change the Swagger, but to tell the linter to surpress the warnings because it's too late to change the server.

TL;DR; If this is output from Azure, and if you return "DPMServers", it won't deserialize with "dpmServers".

Reply:
In this case, server will return "dpmServers". Though old SDKs will be broken then.

Regarding #1, I'll call that a feature as that is the common implementation across libraries is to be case insensitive. Since Python json library, deserializes into dict and not class instance, that creates problem in python (standard implementation).

Reply:
Maybe it's common in the C# world to consider JSON case-insensitive, but not in other language :). I insist on others, it's Python, NodeJS and more language.
Note too that it's a requirement of Swagger:
https://swagger.io/specification/#format-11
- All field names in the specification are case sensitive.

Reply:
So other than this, PR looks good to me for merge. **Tests are failing which I'll update in separate PR.**

Regarding deserialization of JSON in rest api - I'm speaking with my experience on Newtonsoft, Golang std JSON unmarshalling and Python framework Django-Rest-Framework. They all have fallback from case-sensitivity to case-insensitivity if there is no exact case match.

Reply:
@DheerendraRathor I can't merge this PR with the broken tests, this would break my "master" CI. I gave you permissions to push in that branch:
https://github.com/AutorestCI/azure-sdk-for-python/invitations

You can also work from your fork and cherry-pick the generation from here, at your convenience.